LICENSE-libmv.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2007-2011 libmv authors.
+Copyright (c) 2010 libmv authors.
```

Comment:
Please explain me why you are taking the right to change the date.

Reply:
It's a manipulation error I made in the renaming process.
This file has been modified accidentally and I took the text back from an header file, and didn't notice that it was not exactly the same than the global file. No idea, why I have not just use "git checkout" to revert it!
Thanks for pointing that, I fixed it.